docs/src/content/en/docs/deployment/serverless-platforms/cloudflare-deployer.mdx

```diff
@@ -0,0 +1,102 @@
+---
+title: "Cloudflare Deployer"
+description: "Learn how to deploy a Mastra application to Cloudflare using the Mastra CloudflareDeployer"
+---
+
+import { FileTree } from "nextra/components";
+
+# CloudflareDeployer
+
+The `CloudflareDeployer` class handles deployment of standalone Mastra applications to Cloudflare Workers. It manages configuration, deployment, and extends the base [Deployer](/reference/deployer/deployer) class with Cloudflare specific functionality.
+
+## Installation
+
+```bash copy
+npm install @mastra/deployer-cloudflare@latest
+```
+
+## Usage example
+
+```typescript filename="src/mastra/index.ts" showLineNumbers copy
+import { Mastra } from "@mastra/core/mastra";
+import { CloudflareDeployer } from "@mastra/deployer-cloudflare";
+
+export const mastra = new Mastra({
+  // ...
+  deployer: new CloudflareDeployer({
+    projectName: "hello-mastra",
+    scope: "",
+    auth: {
+      apiToken: process.env.CLOUDFLARE_API_TOKEN!,
+      apiEmail: "name@email.com"
+    }
+  })
+});
+```
+
+> See the [CloudflareDeployer](/reference/deployer/cloudflare) API reference for all available configuration options.
+
+### Cloudflare API token
+
+In your Cloudflare dashboard, navigate to **Profile** > **API Tokens**, then select **Edit Cloudflare Workers** and click **Use template** to create a token for the `CloudflareDeployer`.
+
+> To learn how to create a Cloudflare API token, see [Create API token](https://developers.cloudflare.com/fundamentals/api/get-started/create-token/) in the Cloudflare docs.
+
+## Continuous integration
```

Comment:
@adeniyii, hey, here's what i've tried. Builds fail. I've Slacked you the logs. is it a config setting, or something to do with the Cloudflare Deployer? 

I can remove this section if it's something that will take a while to investigate?